src/MatrixTypes_Native.f90

```diff
@@ -663,7 +659,7 @@ SUBROUTINE get_DenseSquareMatrixType(matrix,i,j,getval)
 !>
 !> This routine gets the values of the sparse matrix.  If an (i,j) value
 !> is not present, then 0.0 is returned.  If the (i,j) location is out of
-!> bounds, then -1051.0 is returned (-1051.0 is an arbitrarily chosen key).
+!> bounds, then an exception is raised.
```

Comment:
might need to update this comment since exceptions were removed

Reply:
Say code crashes if DBC enabled... or just remove the whole sentence and expect the dev not to use out of bounds?

Reply:
I personally prefer the first

Reply:
Okay fixed.